app/assets/javascripts/Components/Result/submission_selector.jsx

```diff
@@ -113,8 +113,8 @@ export class SubmissionSelector extends React.Component {
               value={this.props.num_marked}
               min={0}
               max={this.props.num_assigned}
-              low={this.props.num_assigned * 0.35}
```

Comment:
I think the check rules out 0 as a possible value, but we should probably allow 0 (and that should also be the default, not 1)